server/forge/github/parse.go

```diff
@@ -174,8 +174,9 @@ func parsePullHook(hook *github.PullRequestEvent, merge bool) (*github.PullReque
 			hook.GetPullRequest().GetHead().GetRef(),
 			hook.GetPullRequest().GetBase().GetRef(),
 		),
-		PullRequestLabels: convertLabels(hook.GetPullRequest().Labels),
-		FromFork:          fromFork,
+		PullRequestLabels:    convertLabels(hook.GetPullRequest().Labels),
+		PullRequestMilestone: hook.GetPullRequest().GetMilestone().GetTitle(),
```

Comment:
Won't this break if there is no milestone? 🤔

Reply:
no tjats why i use the func of the sdk ... it checks if itself is nil and returns nil the last func checks if nil and returns "" else the milestone value ... :)